app/__init__.py

```diff
@@ -29,6 +29,14 @@
 
 fellows_list = ["Aerin", "EXAMPLE", "SAMPLE"]
```

Comment:
A small nitpick, with the new about.json data I think we can remove the fellows_list variable here